app/src/main/res/values/strings.xml

```diff
@@ -174,6 +174,12 @@
     <string name="hide_soft_when_physical_off_summary">Show soft keyboard when physical keyboard is
         used
     </string>
+    <string name="enable_alt_space_language_shortcut">Alt + Space switches languages</string>
+    <string name="enable_alt_space_language_shortcut_on_summary">Alt + space switches to the next available layout.</string>
```

Comment:
Maybe indicate that this is affecting only physical keyboards?
`Hardware ALT + SPACE switches to the next available layout.`

Reply:
It is in the "Physical Keyboard Tweaks" section, but the headers are fairly small so I think that makes sense.